bin/apisix

```diff
@@ -200,10 +204,14 @@ http {
 
     include mime.types;
 
-    real_ip_header {* real_ip_header *};
-    {% for _, real_ip in ipairs(real_ip_from) do %}
+    {% if http.real_ip_header then %}
```

Comment:
Is this a modification that is not backward compatible?

Reply:
Yes. I will add it.